applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_node_utilities.cpp

```diff
@@ -78,4 +93,42 @@ KRATOS_TEST_CASE_IN_SUITE(NodeUtilities_UpdatesEverythingWhenNoComponentIsFixed,
     KRATOS_EXPECT_VECTOR_NEAR(expected_vector, node.FastGetSolutionStepValue(ACCELERATION, 0), 1e-12)
 }
 
+KRATOS_TEST_CASE_IN_SUITE(AssignUpdatedVectorVariableToNodes_UpdatesEverythingRegardlessOfFixes,
```

Comment:
Question: 'Fixities' rather than 'Fixes'?

```suggestion
KRATOS_TEST_CASE_IN_SUITE(AssignUpdatedVectorVariableToNodes_UpdatesEverythingRegardlessOfFixities,
```

Reply:
done